engine/execution/checker/core.go

```diff
@@ -0,0 +1,176 @@
+package checker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/engine/execution/state"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol"
+	"github.com/onflow/flow-go/storage"
+)
+
+// trigger the check once every minute,
+const DefaultTimeInterval = time.Minute * 1
```

Comment:
```suggestion
// DefaultTimeInterval triggers the check once every minute,
const DefaultTimeInterval = time.Minute * 1
```
This belongs in the `engine.go` I think.